src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/theme-switcher.html

```diff
@@ -3,9 +3,9 @@
 <script>
 document.write(`
   <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="{{ _('light/dark') }}" aria-label="{{ _('light/dark') }}" data-bs-placement="bottom" data-bs-toggle="tooltip">
-    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light"></i>
-    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark"></i>
-    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"></i>
+    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light" title='Current theme: light'></i>
+    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark" title='Current theme: dark'></i>
+    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto" title='Current theme: follows OS Default'></i>
```

Comment:
Double quotes and translatable strings

```suggestion
    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light" title="{{ __('Current theme: light') }}"></i>
    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark" title="{{ __('Current theme: dark') }}"></i>
    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto" title="{{ __('Current theme: follows OS Default') }}"></i>
```

Reply:
it looks like you're using double-underscores for the translatable string, I thought it was single underscore?  Is that a Jinja-specific thing?

Reply:
Oh that's my bad. Should be single then